src/index.test.ts

```diff
@@ -63,6 +64,77 @@ describe("voices", () => {
 });
 
 describe("tts.websocket", () => {
+  let phonicWebSocket: PhonicWebSocket;
+
+  /**
+   * Promise that resolves with all messages received from the websocket after a period of no messages. Can be awaited
+   * multiple times.
+   */
+  let allMessagesReceived: Promise<PhonicWebSocketResponseMessage[]> | null =
+    null;
+  const maxIdleTime = 5000; // If we don't receive messages for 5 seconds, we consider that we received all of them.
+
+  const safeGet = (
+    messages: PhonicWebSocketResponseMessage[] | null,
+    idx: number,
+  ) => {
+    if (messages === null) {
+      throw new Error("messages is null");
+    }
+    if (messages[idx] === undefined) {
+      throw new Error(`Expected message at index ${idx} but received less`);
+    }
+    return messages[idx];
+  };
+
+  beforeEach(async () => {
+    const phonic = new Phonic(apiKey, { baseUrl });
+    const { data, error } = await phonic.tts.websocket();
+
+    if (error !== null) {
+      throw new Error("Failed to connect to websocket");
+    }
+
+    phonicWebSocket = data.phonicWebSocket;
+
+    const messages: PhonicWebSocketResponseMessage[] = [];
+    let messagesTimeoutId: NodeJS.Timer | null = null;
+    let allMessagesReceivedResolve:
+      | ((messages: PhonicWebSocketResponseMessage[]) => void)
+      | null = null;
+
+    const createNewPromise = () => {
+      messages.length = 0; // Clear previous messages
```

Comment:
Let's do the more typical `messages = []` to allow it to be garbage collected.
Need to define `messages` as `let` for that.

Reply:
Done